### DISCUSSION TITLE
Error: Analyze the lobster outputs

### DISCUSSION THREAD

**Discussion by irfanulas** [JaGeo/LobsterPy#327] [General]
Hello,
I am getting error indicating ' ValueError: could not convert string to float: 'for' ' when I run _lobsterpy description_ command. The version of lobster used is LOBSTER 5.1.0 and the DFT package is QuantumEspresso 7.2.  Unfortunately I could not figure out the source of the error.
Thanks

**Comment by JaGeo** [comment]
Thanks. @naik-aakash and I will look into it soon.

**Reply by irfanulas** [reply]
Thanks for the quick response.

**Reply by JaGeo** [reply]
@irfanulas We have now released a new lobsterpy version that can be used with LOBSTER 5.1.0. I hope this solves your issue.
